packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -79,65 +81,34 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
   }
 
   constructor(
-      onUpdate: () => void, gltf: GLTF, gltfMaterial: GLTFMaterial,
-      gltfIndex: number, isActive: boolean,
+      onUpdate: () => void,
+      gltfIndex: number,
+      isActive: boolean,
       modelVariants: Map<string, VariantData>,
       correlatedMaterials: Set<MeshPhysicalMaterial>,
-      lazyLoadInfo: LazyLoader|undefined = undefined) {
-    super(onUpdate, gltfMaterial, correlatedMaterials);
+      name: string|undefined,
+      lazyLoadInfo: LazyLoader|undefined = undefined,
+  ) {
+    super(onUpdate, correlatedMaterials);
     this[$gltfIndex] = gltfIndex;
     this[$isActive] = isActive;
     this[$modelVariants] = modelVariants;
+    this[$name] = name;
 
     if (lazyLoadInfo == null) {
-      this[$initialize](gltf);
+      this[$initialize]();
     } else {
       this[$lazyLoadGLTFInfo] = lazyLoadInfo;
     }
   }
 
-  private[$initialize](gltf: GLTF): void {
+  private[$initialize](): void {
     const onUpdate = this[$onUpdate] as () => void;
-    const gltfMaterial = this[$sourceObject] as GLTFMaterial;
     const correlatedMaterials =
         this[$correlatedObjects] as Set<MeshPhysicalMaterial>;
 
-    if (gltfMaterial.extensions &&
-        gltfMaterial.extensions['KHR_materials_pbrSpecularGlossiness']) {
-      console.warn(`Material ${gltfMaterial.name} uses a deprecated extension
-          "KHR_materials_pbrSpecularGlossiness", please use
-          "pbrMetallicRoughness" instead. Specular Glossiness materials are
-          no longer supported; to convert to metal-rough, see 
-          https://www.donmccurdy.com/2022/11/28/converting-gltf-pbr-materials-from-specgloss-to-metalrough/.`);
```

Comment:
I'm a little sad to loose this warning, but I think it's okay because three.js throws its own anyway.

Reply:
RIP